neo4j/v1/routing.py

```diff
@@ -140,6 +143,9 @@ def update(self, new_routing_table):
         self.last_updated_time = self.timer()
         self.ttl = new_routing_table.ttl
 
+    def servers(self):
+        return set(self.routers.elements() + self.writers.elements() + self.readers.elements())
```

Comment:
This can just be `return set(self.routers) | set(self.writers) | set(self.readers)` and there is no need to add an `elements` accessor on `OrderedSet`.